main.cc

```diff
@@ -691,7 +691,7 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             // disable reactor stall detection during startup
             auto blocked_reactor_notify_ms = engine().get_blocked_reactor_notify_ms();
             smp::invoke_on_all([] {
-                engine().update_blocked_reactor_notify_ms(std::chrono::milliseconds(1000000));
+                engine().update_blocked_reactor_notify_ms(std::chrono::milliseconds(1000000000));
```

Comment:
Can we use something more readable like `hours(1)`?

Reply:
Or `1h`

Reply:
milliseconds(1000000) could be written 1000s, no need to cast.

Reply:
done